wiki/Tournaments/TMC/4th/en.md

```diff
@@ -0,0 +1,234 @@
+| Statistician | ::{ flag=US }:: [EpsilonMaiagare](https://osu.ppy.sh/users/3855052), ::{ flag=CN }:: [Rush_FTK](https://osu.ppy.sh/users/3046856), ::{ flag=CN }:: [YuEast 2018](https://osu.ppy.sh/users/13953619) |
+| Wiki editor | ::{ flag=US }:: [akace100](https://osu.ppy.sh/users/9308128), ::{ flag=CN }:: [Rush_FTK](https://osu.ppy.sh/users/3046856), ::{ flag=CN }:: [YuEast 2018](https://osu.ppy.sh/users/13953619) |
+| Designer | ::{ flag=CN }:: [[GB]ChickenGold](https://osu.ppy.sh/users/16586663), ::{ flag=MY }:: [lous](https://osu.ppy.sh/users/6231292), ::{ flag=CN }:: [lovely_hyahya](https://osu.ppy.sh/users/10318380), ::{ flag=CN }:: [Sakura006](https://osu.ppy.sh/users/10365024), ::{ flag=CN }:: [YuEast 2018](https://osu.ppy.sh/users/13953619) |
+
+## Links
+
+- **[Information spreadsheet](https://docs.google.com/spreadsheets/d/11iFCHTWJLZhRDrtsDUjMx8hwKGWUYu0VR_x3SDgTOS0)**
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/2015815)
+- [THMC 4 Discord server](https://discord.gg/cAr5VHG7WX)
+- [Livestream](https://www.twitch.tv/touhoumaniacup_official)
+- [Challonge bracket](https://challonge.com/)
+- [Liquipedia](https://liquipedia.net/osu/Touhou_Project_Mania_Cup/4)
+
+<!--## Participants
+
+## Groups 
+
+## Podium
+
+## Mappools-->
+
+## Ruleset
+
+The ruleset is primarily based on rulesets used in the [osu!mania 4K World Cup 2024](/wiki/Tournaments/MWC/2024_4K#ruleset) and [4 Digit osu!mania World Cup 2024](/wiki/Tournaments/4DM/2024#ruleset).
+
+- Any general rules not shown below but listed in the reference tournament are still valid.
+
+### Tournament rules
+
+1. The Touhou Project Mania Cup 4th (THMC 4th) is a 2 versus 2 double-elimination team tournament, played on the osu!mania game mode (4K variant).
+2. If no referee is available at match time, the match will be postponed to a later time.
+3. Failed players' scores will still count towards team scores.
+4. Use of visual settings to alter background dim or disable beatmap elements like storyboards and skins are allowed.
```

Comment:
```suggestion
4. Use of visual settings to alter background dim or disable beatmap elements like storyboards and skins is allowed.
```

Reply:
I'm not sure since MWC 7K 2025 wiki also using that
![QQ_1734419814746](https://github.com/user-attachments/assets/c59bcdc6-2e76-4f3a-9048-c3d1d117aded)
waiting for response, same with other mirror adjustment
if no future discussion on links ready , i will accept those suggestion now

Reply:
yeah i feel like are is better but ran thru grammar checking tool to make sure and its saying "is" better